pyproject.toml

```diff
@@ -51,7 +51,7 @@ start = "python manage.py run --debug"
 makemigrations = "python manage.py makemigrations"
 django_shell = "python manage.py shell"
 test = "coverage run manage.py test"
-coverage = "coverage run manage.py test --no-input; coverage report -m"
+coverage = "coverage run manage.py test --no-input && coverage report -m"
```

Comment:
Would be nice if this ran migrations too.

Reply:
That sounds fine. Would you prefer the initial approach of putting it in the manage.py, or should we do it here?

Reply:
Doing it in `manage.py` would be better so it still works without reliance on Poetry's task.

Reply:
And you could also avoid useless stuff like collecting static files

Reply:
Added in 98fa159